libs/string/src/lib.sw

```diff
@@ -20,13 +27,24 @@ impl String {
         self.bytes.clear()
     }
 
+    /// Converts bytes to a `String`.
+    ///
+    /// # Arguments
+    ///
+    /// * `bytes` - The bytes which will be converted into a `String`.
+    pub fn from_bytes(bytes: Bytes) -> String {
+        String { bytes }
+    }
+
     /// Converts a vector of bytes to a `String`.
     ///
     /// # Arguments
     ///
     /// * `bytes` - The vector of `u8` bytes which will be converted into a `String`.
-    pub fn from_utf8(bytes: Vec<u8>) -> String {
-        String { bytes }
+    pub fn from_utf8(mut bytes: Vec<u8>) -> String {
+        String {
+            bytes: Bytes::from_vec_u8(bytes),
+        }
```

Comment:
```suggestion
    pub fn from_utf8(mut bytes: Vec<u8>) -> Self {
        Self {
            bytes: Bytes::from_vec_u8(bytes),
        }
```

Does this need to be `mut`?

Reply:
Yes, without it I get a `Cannot call method "from_vec_u8" on variable "bytes" because "bytes" is not declared as mutable.` as it's defined as mutable in the Bytes library. 

cc @nfurfaro